nemoguardrails/library/domain_hallucination/files/03_scripts/generate_report.py

```
@@ -0,0 +1,270 @@
+    L("|----------|-----------|--------|-----|---------|")
+    for label, m in sorted(overall.get("per_decision", {}).items()):
+        L(f"| {label} | {m['precision']:.4f} | {m['recall']:.4f} "
+          f"| {m['f1']:.4f} | {m['support']} |")
+    L("")
+
+    # ── Confusion Matrix
+    cm = overall.get("confusion_matrix", {})
+    if cm:
+        all_labels = sorted(set(
+            list(cm.keys()) +
+            [l for row in cm.values() for l in row.keys()]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)